.github/workflows/run-tests.yml

```diff
@@ -10,12 +10,12 @@ jobs:
     container: python:3.8
 
     services:
-      postgres:
+      db:
         image: postgres:latest
         env:
-          POSTGRES_DB: postgres
+          POSTGRES_DB: test
```

Comment:
Won't these new values break the database URL shown below?

Reply:
Looks like it might...

Reply:
I'm changing these values to show that we can have different database names etc as long as we define `DATABASE_URL` (on line 37).

Reply:
(the github run-test action is still passing, which should indicate that this is working?)